docs/photoniq/vwrs/configure-vwrs-priority.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 30
+title: Configure a VWRs Priority Waiting Room
+---
+
+Priority waiting rooms allow you to prioritize traffic based on the priority of the request. Requests are assigned a priority level and then dequeued at different rates based on the priority level. This feature helps handle various types of traffic, such as high-priority, regular, and low-priority.
+
+VWRs dequeues requests by assigning weights to different priority levels descendingly, with the weight halving for each subsequent level. For instance, if there are three priority levels, the highest priority receives a weight of 4, the next lower priority gets a weight of 2, and the lowest priority is assigned a weight of 1. This creates a total weight summing up to 7 for all priorities combined. A formula determines the distribution of dequeued items from each priority level: RateLimit \* (Weight of Specific Priority Level / Total Weight of All Priorities). If a priority level has no requests to dequeue, its weight is redistributed among the remaining priorities, recalculating the weights based on the active queues. Additionally, if there are any remaining items after distribution, those are allocated to the lowest priority level.
```

Comment:
This is great content. If you wanted to get more into priorities, we could write a concept topic similar to [VWRs Queue Types](https://www.macrometa.com/docs/photoniq/vwrs/queue-types).

Reply:
Priority queues are an optional configuration.  My guess would be 80+% will never use priority queues.  So thought we would pull it out of getting started because you don't need it to getting started.

Priority queues are not a different queue type.  It is configured on top of the queue type.  So a FIFO queue can have priority capabilities configured and can any of the other queue types.  You can never configure a priority queue by itself.  It needs to be part of the queue type.